framework/src/headers/ip/v6/icmp/ndp.rs

```diff
@@ -0,0 +1,124 @@
+use std::default::Default;
+use std::fmt;
+use std::marker::PhantomData;
+use std::net::Ipv6Addr;
+
+use headers::{CalcChecksums, EndOffset, Ipv6VarHeader};
+
+use super::{IcmpMessageType, Icmpv6Header};
+
+#[derive(Debug)]
+#[repr(C)]
+pub struct NdpMessage<T>
```

Comment:
we can remove this entire thing. this should have been cleaned up when i was removing neighbor messages.